dask/dataframe/core.py

```diff
@@ -810,6 +810,34 @@ def min(self, axis=None, skipna=True):
                                  skipna=skipna, axis=axis,
                                  meta=meta)
 
+    def idxmaxmin(self, fn, axis=None, skipna=True):
+        if fn == 'idxmax':
+            idxfn = _idxmax
+        elif fn == 'idxmin':
+            idxfn = _idxmin
+        else:
+            raise ValueError('Function must be idxmax or idxmin')
+
+        axis = self._validate_axis(axis)
+        # if axis == 1:
+            # return map_partitions(idxfn, None, self,
+                                  # token=self._token_prefix + fn,
+                                  # skipna=skipna, axis=axis)
+        # else:
```

Comment:
I may have my knowledge of axes swapped around, but I think that one of these choices only needs to look at a row at a time.  In this case (whichever it is) then yes I think that `map_partitions` should be sufficient.  This is far nicer than aca as well because we'll still have all of our partitions rather than just a single giant pandas dataframe